contracts/MANAContinuousSale.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.4.11;
+
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./ContinuousSale.sol";
+
+contract MANAContinuousSale is ContinuousSale, Ownable {
+
+    uint256 public constant INFLATION = 8;
+
+    bool public started = false;
```

Comment:
I did not want to obscure too much the logic with an inheritance chain, but it's worth considering.